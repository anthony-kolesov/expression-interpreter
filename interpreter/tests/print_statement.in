# Valid string.
print "pi = \n"
# Another valid string.
print "foobar"
# identifiers are not accepted
print foobar
# numbers are not accepted
print 4
print 4.5
