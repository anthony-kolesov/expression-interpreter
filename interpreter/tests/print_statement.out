pi = 
foobarERROR:6,6-6,11:syntax error, unexpected TOKEN_IDENTIFIER, expecting TOKEN_STRING
ERROR:8,6:syntax error, unexpected TOKEN_NUMBER, expecting TOKEN_STRING
ERROR:9,6:syntax error, unexpected TOKEN_NUMBER, expecting TOKEN_STRING
